src/components/Link/Link.tsx

```diff
@@ -27,18 +26,22 @@ const styleDark = css`
   }
 `;
 
-interface LinkProps {
-  children: React.ReactChild;
-  to?: string;
+interface Props {
+  children: React.ReactNode;
+  to: string;
   dark?: boolean;
+  rel?: string;
+  target?: string;
 }
 
-const Link = ({
+const Link: React.FunctionComponent<Props> = ({
   children,
-  to = '#',
+  to,
   dark = false,
-}: LinkProps): React.ReactElement => (
-  <a css={dark ? styleDark : styleLight} href={to}>
+  rel = undefined,
+  target = undefined,
```

Comment:
Hi @alihamoody  - I think it may be better for `rel` and `target` to be covered by using the object spread operator to propagate `props.rest` through to the rendered element:


```es6
const Link: React.FunctionComponent<Props> = ({
  children,
  to,
  dark = false,
  ...rest,
}: Props): React.ReactElement => (
 <a css={etc} ...rest>

```

Reply:
Done for `rel` and `target`
Can't do for `href` because the prop name is different.